### DISCUSSION TITLE
Firmware version 3.0.13

### DISCUSSION THREAD

**Discussion by Bolten88** [fabianishere/udm-iptv#205] [Q&A]
Has someone tested the helper tool with the latest firmware of the UDM se?

**Comment by VentingBow5** [comment]
I upgraded this afternoon and after months without issues I now get the following stb-nmc-400 error with KPN iTV.

**Reply by Bolten88** [reply]
I just installed it. I had to reinstall the tool but then it worked as expected.

**Reply by VentingBow5** [reply]
Tried this as well but did not solve my problem.

**Reply by sbo77** [reply]
I'm getting errors while installing,  is this the same issue? 
Unpacking udm-iptv (2.1.4) ...
dpkg: warning: udm-iptv: unable to stat config file '/etc/udm-iptv.conf'
 (= '/mnt/persistent/udm-iptv.conf'): Too many levels of symbolic links
Setting up igmpproxy (0.3-1) ...
Setting up udm-iptv (2.1.4) ...
dpkg: warning: udm-iptv: unable to stat config file '/etc/udm-iptv.conf'
 (= '/mnt/persistent/udm-iptv.conf'): Too many levels of symbolic links
dpkg: warning: udm-iptv: unable to open /mnt/persistent/udm-iptv.conf for hash: Too many levels of symbolic links
dpkg: error processing package udm-iptv (--configure):
 unable to stat new distributed conffile '/mnt/persistent/udm-iptv.conf.dpkg-new': Too many levels of symbolic links
Errors were encountered while processing:
 udm-iptv
W: Repository is broken: udm-iptv:arm64 (= 2.1.4) has no Size information
E: Sub-process /usr/bin/dpkg returned an error code (1)

**Reply by fabianishere** [reply]
Remove the symlink:

```
rm /etc/udm-iptv.conf
```

**Reply by sbo77** [reply]
thanks!! i will try
